dashboard/app/dsl/user_id_replace_dsl.rb

```diff
@@ -0,0 +1,2 @@
+class UserIdReplaceDSL < ExternalDSL
+end
```

Comment:
It's reasonable to follow the existing pattern in `ExternalDSL`, but given that we're trying to move away from `DSLDefined` I wonder if it makes sense to use the `.level` format?  Same feedback as here: https://github.com/code-dot-org/code-dot-org/pull/13833#discussion_r106543404